data/database.py

```diff
@@ -2022,7 +2022,7 @@ class OrganizationRhSkus(BaseModel):
     RH subscriptions
     """
 
-    subscription_id = IntegerField(index=True, unique=True)
+    subscription_id = IntegerField(index=True)
     user_id = ForeignKeyField(User, backref="org_bound_subscription")
     org_id = ForeignKeyField(User, backref="subscription")
```

Comment:
Note: when defining peewee models, you don't need `_id` at the end of field names. Peewee allows you to access the id by `<field_name>_id`. In this case, if you access `user_id`, will that return a reference to the other model, or the actual db id? e.g. `org_sku.user_id`